### PR TITLE
🐛 Destination Vectara: Improve title extraction logic

### PR DESCRIPTION
**Pull request by ofermend**
## What
* extraction of title in Vectara connector had a bug. It used dpath.util.get() instead of dpath.util.values()

## How
* updated the code to use dpath.util.values()

### CODE REVIEW

**Comment by marcosmarxm** [comment]
Thanks @ofermend at the moment the connector integration tests are working. I'll run your change to check if they return to a green status.

File: airbyte-integrations/connectors/destination-vectara/destination_vectara/writer.py [hunk 1]
```diff
@@ -105,7 +105,7 @@ def _get_document_metadata(self, record: AirbyteRecordMessage) -> Dict[str, Any]
     def _get_document_title(self, record: AirbyteRecordMessage) -> str:
         title = "Untitled"
         if self.title_field:
-            title = dpath.util.get(record.data, self.title_field)
+            title = dpath.util.values(record.data, self.title_field, separator=".")[0]
```
Comment (aaronsteers) [inline airbyte-integrations/connectors/destination-vectara/destination_vectara/writer.py:108]: Is it possible that the call to `values()` would return zero results, for instance if `title_field` is not found?

If that's feasible (and I'm not sure), then what do you thin of a quick check first? 

Something like...

```suggestion
            found_title = dpath.util.values(record.data, self.title_field, separator=".")
            if found_values:
                title = found_title[0]
```

**Comment by gHashTag** [comment]
Approve please

**Comment by marcosmarxm** [comment]
@ofermend can you check AJ last comment? Thanks!

**Comment by ofermend** [comment]
Oh sorry I missed this - yes, this looks like a great improvement. Let's use this suggestion.

**Comment by gHashTag** [comment]
Look forward to

**Review by aaronsteers — APPROVED** [review]
Thanks for the update - and my apologies for letting this go stale.

**Pull request merged**